docs/developer/procedures/supported.rst

```diff
@@ -30,18 +30,18 @@ x      Ask                           Ask to be included in the next release
 Visibility / Module Status
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The GeoTools Module Matrix defines a couple quick QA tests allowing you to rate your module according to the number of gold stars it has earned.
+The GeoTools Gold Star Quality Assurance Check defines a couple quick QA tests allowing you to rate your module according to the number of gold stars it has earned.
 
 1. The first step is to rate your module:
    
    :doc:`check`
 
 2. GeoTools 2.5 and onward expects four stars for a supported module.
 
-3. Create a Module Matrix page:
+3. Create a README.md file:
    
    * show project status
-   * list jira issues (from as many projects as possible)
+   * show IP status
```

Comment:
Yes, e.g. https://github.com/geotools/geotools/blob/master/modules/extension/wms/README.md

I'll add an example to the unsupported/example project